contracts/HolographOperator.sol

```diff
@@ -614,59 +662,13 @@ contract HolographOperator is Admin, Initializable, IHolographOperator {
     return current;
   }
 
-  function getPodBondAmount(uint256 pod) external view returns (uint256 base, uint256 current) {
-    base = _getBaseBondAmount(pod - 1);
-    current = _getCurrentBondAmount(pod - 1);
-  }
-
-  function getBondedPod(address operator) external view returns (uint256 pod) {
-    return _bondedOperators[operator];
-  }
-
-  // add top-up option
-
-  function unbondUtilityToken(address operator, address recipient) external {
-    require(_bondedOperators[operator] != 0, "HOLOGRAPH: operator not bonded");
-    if (msg.sender != operator) {
-      require(_isContract(operator), "HOLOGRAPH: operator not contract");
-      // check that operator is ownable contract
-      require(Ownable(operator).isOwner(msg.sender), "HOLOGRAPH: sender not owner");
-    }
-    address utilityToken = IHolographRegistry(_registry()).getUtilityToken();
-    uint256 amount = _bondedAmounts[operator];
-    // here we subtract our fee for unbonding
-    require(ERC20Holograph(utilityToken).transfer(recipient, amount), "HOLOGRAPH: token transfer failed");
-    //// we need to track operator pod index for easy removal
-    // _popOperator(_bondedOperators[operator] - 1, operatorPodIndex);
-    _bondedOperators[operator] = 0;
-    _bondedAmounts[operator] = 0;
-  }
-
-  function bondUtilityToken(
-    address operator,
-    uint256 amount,
-    uint256 pod
-  ) external {
-    require(_bondedOperators[operator] == 0, "HOLOGRAPH: operator is bonded");
+  function _RBH(
```

Comment:
RBH is random block hash?

Reply:
Yes, I wrote it shorthand at that point due to time constraints, will rename it to make more clear in next iteration.

Reply:
cool